integrations/api_packages_test.go

```diff
@@ -100,3 +104,26 @@ func TestPackageAPI(t *testing.T) {
 		MakeRequest(t, req, http.StatusNoContent)
 	})
 }
+
+func TestPackageCleanup(t *testing.T) {
+	defer prepareTestEnv(t)()
+
+	time.Sleep(time.Second)
```

Comment:
Is this some kind of bug or behavior issue?

Reply:
That's only for the test to ensure a different timestamp.

Reply:
As that test will be executed quite frequently, I'd propose to rather use a value such as
```suggestion
	// Ensure that the timestamp will be different
	time.Sleep(10 * time.Millisecond)
```

Reply:
But the timestamps in the database are in seconds so it needs to be a `Second`. Why do you think it will be executed quite frequently? It only runs once.

Reply:
> But the timestamps in the database are in seconds so it needs to be a Second

Ah okay, yeah, that makes sense then. I thought you simply wanted to ensure that you would get another timestamp when creating a new one.

> It only runs once

Once per database for every commit. That was why I wanted to shorten the delay.